rainforest-cli.go

```diff
@@ -8,25 +8,33 @@ import (
 
 var apiToken string
 
+var smartFolderID int
+var siteID int
+var tags string
+
 var baseURL = "https://app.rainforestqa.com/api/1"
 var out io.Writer = os.Stdout
 
-func parseCommands() []string {
+func parseCommands(arguments []string) []string {
 	commands := make([]string, 0, 5)
-	for i := len(os.Args) - 1; i > 0; i-- {
-		if os.Args[i][0] != '-' {
-			commands = append(commands, os.Args[i])
-			os.Args = append(os.Args[:i], os.Args[i+1:]...)
+	for i := len(arguments) - 1; i > 0; i-- {
+		if arguments[i][0] != '-' {
+			commands = append(commands, arguments[i])
+			os.Args = append(arguments[:i], arguments[i+1:]...)
 		}
 	}
 	return commands
 }
 
 func main() {
-	commands := parseCommands()
+	commands := parseCommands(os.Args)
 	command := commands[0]
 
-	flag.StringVar(&apiToken, "token", "", "API token. You can find your account token at https://app.rainforestqa.com/settings/integrations")
+	flag.StringVar(&apiToken, "token", apiToken, "API token. You can find your account token at https://app.rainforestqa.com/settings/integrations")
```

Comment:
In our other Go projects we have flag vars as global variable, which I think looks cleaner. I.e. instead of this, you'd have a top level variable: 

```go
var apiToken= flag.String("token", "", ...)
```